cms/templates/cms/home_page.html

```diff
@@ -160,5 +177,90 @@
   </div>
 
 </div>
+
+<!-- Modal subscribe alerts -->
+<div id="subscribeAlert" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">Sign up to be notified when new programs are announced.</h4>
+      </div>
+      <div class="modal-body">
+        <div class="container">
+            <form class="form-horizontal subscription-form mailchimp">
+                <!-- SUBSCRIPTION SUCCESSFUL OR ERROR MESSAGES -->
+                <h4 class="col-sm-offset-4 subscription-success"></h4>
+                <h4 class="col-sm-offset-4 subscription-error"></h4>
+
+                <div class="form-group">
+                    <label class="col-sm-4 control-label">Name</label>
+                    <div class="col-sm-8">
+                        <input type="text" class="form-control" id="inputname" name="FNAME">
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-sm-4 control-label">Email</label>
+                    <div class="col-sm-8">
+                        <input type="email" class="form-control" id="inputEmail3" placeholder="">
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-sm-4 control-label">Tell us more...</label>
+                    <div class="col-sm-8">
+                        <div class="radio">
+                            <label><input type="radio" name="MMERGE2" value="I am an interested learner" id="learner">
```

Comment:
This mailchimp code is copied from existing app see this please https://github.com/mitodl/micromasters/issues/287#issuecomment-219832719